### ISSUE TITLE
Quiet mode through .condarc

### ISSUE THREAD

**Issue by jaimergp** [conda/conda#10811]
## Current Behavior

`.condarc` allows to specify default configuration settings that otherwise are set up via command line flags. For example, `--always-yes`. Surprisingly, while there's a flag to [set `quiet` for `conda-build`](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#quiet), there's no equivalent for `conda` itself. If it already exists, it's not documented.

## Expected Behavior

I would like a `quiet` setting for `conda` core commands.

**Comment by jaimergp** [comment]
Oh, sorry, it appears to be a documentation issue. `quiet` is indeed mentioned [here](https://docs.conda.io/projects/conda/en/latest/configuration.html), but not [here](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html).

**Comment by jaimergp** [comment]
Not stale. This is a documentation request.

**Comment by jakirkham** [comment]
Do see `quiet` in [this `condarc` config with comments in the docs]( https://docs.conda.io/projects/conda/en/latest/configuration.html )

AFAICT it does work

Possibly it was introduced in PR: https://github.com/conda/conda/pull/2537 ?

Perhaps this simply needs documentation

**Comment by jaimergp** [comment]
Yea I think I opened this because I was confused about the key not being present in https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/settings.html, because I didn't understand that page is not meant to be exhaustive. The autogenerated section you pointed out it _is_ comprehensive, so the duplication across sections is a bit confusing.

**Comment by jakirkham** [comment]
Yeah it is also not the most discoverable section (despite being exhaustive) and lacks anchors for particular values

Do you think the other configuration page should be exhaustive or merely representative (and if the latter what should be included)?

**Comment by jaimergp** [comment]
I opened https://github.com/conda/conda/pull/14435 for now. I think there's another issue that mentions the same problem in the the configuration docs: https://github.com/conda/conda/issues/12355
